Update for titles and short stories on assemblies (#108230)
openshift/openshift-docs: virt/managing_vms/virt-enabling-disabling-vm-delete-protection.adoc

:_mod-docs-content-type: ASSEMBLY
[id="virt-enabling-disabling-vm-delete-protection"]
= Enable or disable virtual machine delete protection
include::_attributes/common-attributes.adoc[]
:context: virt-enabling-disabling-vm-delete-protection

toc::[]

[role="_abstract"]

> Prevent accidental deletion of a virtual machine (VM) by enabling delete protection of your VMs. When you enable delete protection, you are then required to disable it in order to remove the VM.

Reviewer: Possible suggestion to reduce wordiness:

```suggestion
You can prevent accidental deletion of a virtual machine (VM) by enabling delete protection. If delete protection is enabled, you must disable it before you can delete that VM.
```
